samples/subsys/ipc/ipc_service/icmsg_me/CMakeLists.txt

```diff
@@ -6,27 +6,14 @@
 
 cmake_minimum_required(VERSION 3.20.0)
 
-set(REMOTE_ZEPHYR_DIR ${CMAKE_CURRENT_BINARY_DIR}/ipc_service_remote-prefix/src/ipc_service_remote-build/zephyr)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 
-if("${BOARD}" STREQUAL "nrf5340dk_nrf5340_cpuapp")
-  set(BOARD_REMOTE "nrf5340dk_nrf5340_cpunet")
-else()
+if(NOT ("${BOARD}" STREQUAL "nrf5340dk_nrf5340_cpuapp"))
   message(FATAL_ERROR "${BOARD} is not supported for this sample")
 endif()
 
```

Comment:
Would it make sense to make this only buildable using sysbuild?

Reply:
I'm honestly not sure. I mean, it is still possible to build each core image without sysbuild, and I don't know if we win anything by not supporting it. I would lean to keep supporting it.